packages/legend-query/src/components/QueryBuilderSetupPanel.tsx

```diff
@@ -52,6 +56,25 @@ export const QueryBuilderSetupPanel = observer(
     const changeClass = (val: PackageableElementOption<Class>): void => {
       querySetupState.setClass(val.value);
       queryBuilderState.resetData();
+      if (isVersionedClass(val.value)) {
```

Comment:
Could we add a comment here explaining that for now we will default the version value to a parameter. In the future we could default it to something else. Maybe `now()` or `%latest`.